tests/suites/backup_restore/backup_restore_test.go

```diff
@@ -288,7 +288,7 @@ var _ = Describe("backup and restore", func() {
 	})
 
 	// This test is disabled (PIt instead of It) and can be enabled as soon as https://github.com/thelastpickle/cassandra-medusa/pull/119 is merged and released
-	PIt("Creates and restores a backup with JMX SSL and authentication", func() {
+	It("Creates and restores a backup with JMX SSL and authentication", func() {
```

Comment:
This was intentional left as `PIt`. Backup/Restore with SSL doesn't work yet, it's waiting for a new medusa release.

Reply:
Thanks, I thought this was by mistake... that's why the tests fail 😆